### ISSUE TITLE
Version bump for report-uri fix

### ISSUE THREAD

**Issue by kavika13** [rwjblue/ember-cli-content-security-policy#27]
Can you do a version bump so the report-uri fix is in a tag? My console is complaining at me ;)

**Comment by sir-dunxalot** [comment]
You can use the following in your `package.json` until the next release:

```
"ember-cli-content-security-policy": "git://github.com/rwjblue/ember-cli-content-security-policy.git#master",
```

This pulls down the fix for the report-uri errors :)

**Comment by kavika13** [comment]
TY! Good point.

I'll probably do a specific commit id rather than trusting #master :) Ala, https://github.com/bower/bower/issues/856

**Comment by sir-dunxalot** [comment]
@rwjblue any chance of getting the version bump since the patch is already on master?

**Comment by akappen** [comment]
:+1:

**Comment by rwjblue** [comment]
released as 0.4.0

**Comment by sir-dunxalot** [comment]
Thanks, Rob!
